src/components/TxData.js

```diff
@@ -396,10 +409,17 @@ class TxData extends React.Component {
     }
 
     const renderTokenList = () => {
+      const renderTokenUID = (token) => {
+        if (token.uid === hathorLib.constants.HATHOR_TOKEN_CONFIG.uid) {
+          return token.uid;
+        } else {
+          return <Link to={`/token_detail/${token.uid}`}>{token.uid}</Link>
+        }
+      }
       const tokens = this.state.tokens.map((token) => {
         return (
           <div key={token.uid}>
-            <span>{token.name} <strong>({token.symbol})</strong> | {token.uid}</span>
+            <span>{token.name} <strong>({token.symbol})</strong> {token.unknown && <i title='This token is not registered in your wallet.' className='fa text-warning fa-warning'></i>} | {renderTokenUID(token)}</span>
```

Comment:
I think we should add the warning symbol in the inputs and outputs as well. Many users may not pay attention to the token list below the inputs and outputs.

Reply:
All screens have changed a lot. Can you check again, please?